wandb/sdk/launch/builder/build.py

```diff
@@ -92,7 +92,8 @@ def get_docker_user(launch_project: LaunchProject, runner_type: str) -> Tuple[st
 CUDA_SETUP_TEMPLATE = """
 FROM {cuda_base_image} as base
 # TODO: once NVIDIA their linux repository keys for all docker images
-RUN apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/$(cat /etc/os-release | grep ^ID= |  cut -d "=" -f2 )$(cat /etc/os-release | grep ^VERSION_ID= |  cut -d "=" -f2 | sed -e 's/[\".]//g' )/$(uname -i)/3bf863cc.pub
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/$(cat /etc/os-release | grep ^ID= |  cut -d "=" -f2 )$(cat /etc/os-release | grep ^VERSION_ID= |  cut -d "=" -f2 | sed -e 's/[\".]//g' )/$(uname -i)/3bf863cc.pub
+RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/$(cat /etc/os-release | grep ^ID= |  cut -d "=" -f2 )$(cat /etc/os-release | grep ^VERSION_ID= |  cut -d "=" -f2 | sed -e 's/[\".]//g' )/$(uname -i)/7fa2af80.pub
```

Comment:
@KyleGoyette I found [here](https://github.com/wandb/client/pull/3580/files#diff-3e0ea0b64b24d32420132d0d5dd64928b1233eb6311b9467ff2d3db89b00c51e) that this other key makes my image never fail, perhaps it'll help with `mach-launch` too?